dls-controls/malcolmjs
# Updated MaterialUI to v3

## Description

Updated MaterialUI to v3. Also changed disconnected alarm icon to "power_off" material icon (this was not available in the previous version of material UI)

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm run:ui-only and npm run:server (in a separate window)
- navigate to localhost:3000/PANDA
- wait til the site is loaded, then kill the dev server process.
- view the new disconnected icon
